posthog/warehouse/models/external_data_schema.py

```diff
@@ -80,6 +80,31 @@ def sync_old_schemas_with_new_schemas(new_schemas: list, source_id: uuid.UUID, t
         ExternalDataSchema.objects.create(name=schema, team_id=team_id, source_id=source_id, should_sync=False)
 
 
+def get_snowflake_schemas(
+    account_id: str, database: str, warehouse: str, user: str, password: str, schema: str, role: Optional[str] = None
+) -> list[Any]:
+    with snowflake.connector.connect(
+        user=user,
+        password=password,
+        account=account_id,
+        warehouse=warehouse,
+        database=database,
+        schema="information_schema",
```

Comment:
question: Shouldn't this be uppercased? The schema is `INFORMATION_SCHEMA` according to the docs.

Reply:
Ah, nevermind, Snowflake should be auto upper-casing this.